src/pages/FrontPage.scss

```diff
@@ -0,0 +1,83 @@
+@import '../assets/css/fonts';
+
+.content {
+  margin-top: 75px;
+}
+
+.column {
+  display: inline-block;
+  vertical-align: top;
+}
+
+.leftColumn {
+  composes: column;
+  width: 25%;
+}
+
+.rightColumn {
```

Comment:
Det er ingen som trenger å lære noen ting. For ikke så veldig lenge siden kom GitHub med en ny feature som tillater å squashe alle commits i en PR til én melding når den skal merges. Du skal få det valget når du trykker på "Merge pull request".